en_pud-ud-test.conllu

```diff
@@ -16117,8 +16117,8 @@
 6	sought	seek	VERB	VBD	Mood=Ind|Tense=Past|VerbForm=Fin	0	root	0:root	_
 7	for	for	ADP	IN	_	10	case	10:case	_
 8	outgoing	outgoing	ADJ	JJ	Degree=Pos	9	amod	9:amod	_
-9	President	President	PROPN	NNP	Number=Sing	6	obl	6:obl	_
-10	Yudhoyono	Yudhoyono	PROPN	NNP	Number=Sing	9	flat	9:flat	_
+9	President	President	PROPN	NNP	Number=Sing	10	nmod:desc	10:nmod:desc	_
+10	Yudhoyono	Yudhoyono	PROPN	NNP	Number=Sing	6	obl	6:obl	_
```

Comment:
`obl:for`? Although I'm not sure this is the right analysis. It looks like *for* marking the subject of an infinitival clause.

Reply:
Would you be up for editing that tree?  I'm more of a remote terminal than a processor when it comes to annotating dependencies

Reply:
VERB + for-X-to-Y is something of a rare pattern. Updated modeled after this one from EWT:
<img width="866" alt="image" src="https://github.com/user-attachments/assets/290533da-96d3-4f0e-822f-3e8e21745201" />